python/ray/tune/ray_trial_executor.py

```diff
@@ -276,6 +280,10 @@ def reset_trial(self, trial, new_config, new_experiment_tag):
         """
         trial.experiment_tag = new_experiment_tag
         trial.config = new_config
+        if reset_logger is True:
+            trial.reset_logger()
+            new_config['logdir'] = trial.logdir
```

Comment:
do you need this?

Reply:
The `new_config['logdir']` is useful is someone wants to sync the folder remotely inside the `trainable`.

I think it could actually be useful to implement the logic in the base `Trainable class`.
I did not do it yet because I wanted your feedback on this first part

Reply:
Oh I see; I think what you're trying to do is actually necessary. The right way to do this is to separate the Trainable code to have a helper function (`Trainable.create_logger(logger_creator)`), for L78-L8, and then in trial.reset_logger, it also calls `trial.runner.create_logger.remote("some_logger_creator_that_has_the_new_logdir")`

Reply:
This part should be fixed the right way, can you check again?